src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Views/BankId/_LoginScript.cshtml

```diff
@@ -3,8 +3,9 @@
 <script>
     (function (options) {
         // Pre check
+        console.log("checking status");
         var statusElement = document.querySelector('#bankIdLoginStatus');
-        var statusSpinnerElement = statusElement.querySelector('.status-spinner');
```

Comment:
I'd still like to keep the class-name status-spinner, because that's still what it is I think :)